Modules/GLO/include/GLO/MeanVertexPostProcessing.h

```diff
@@ -20,20 +20,37 @@
 #include "QualityControl/PostProcessingInterface.h"
 #include "CCDB/CcdbApi.h"
 
-class TH1F;
+#include <TLine.h>
+#include <TH1F.h>
+#include <TGraph.h>
+#include <TCanvas.h>
 
 namespace o2::quality_control_modules::glo
 {
 
+class TrendGraph : public TCanvas
+{
+ public:
+  TrendGraph(std::string name, std::string title, std::string label, float rangeMin, float rangeMax);
+
```

Comment:
Checking superficially, should work. But the TrendGraph should have a virtual destructor.

Reply:
@shahor02 thanks for checking! The virtual destructor is needed even if it would be empty?